docs/simulate-large-file/api.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"bytes"
+	"net/http"
+	"strconv"
+
+	wasmhttp "github.com/nlepage/go-wasm-http-server"
+)
+
+func main() {
+
+	one_kb := ""
+	for i := 0; i < 1024; i++ {
+		one_kb += "0"
+	}
+
+	http.HandleFunc("/create-file-by-size", func(res http.ResponseWriter, req *http.Request) {
+		vars := req.URL.Query()
+		size, ok := vars["size"]
+		if !ok {
+			bytes.NewBuffer([]byte("url must have size paramter.")).WriteTo(res)
+			return
+		}
+
+		targetSize, _ := strconv.ParseInt(size[0], 0, 0)
```

Comment:
Here you could handle the error in case the size parameter is not a valid number.